Documentation/bash-scripts.md

```diff
@@ -0,0 +1,52 @@
+# Bash scripts
+
+A common way to use acbuild is via a bash script. There's a common template
+that the examples for acbuild follow, and it makes cleanup a little easier. The
```

Comment:
(Either explain how the template "makes cleanup easier", or cut that clause entirely.)